man/systemd-stub.xml

```diff
@@ -65,8 +65,12 @@
       <listitem><para>The initrd will be loaded from the <literal>.initrd</literal> PE section.
       </para></listitem>
 
-      <listitem><para>A compiled binary DeviceTree will be looked for in the <literal>.dtb</literal> PE
-      section.</para></listitem>
+      <listitem><para>Compiled binary DeviceTrees will be looked for in the <literal>.dtb</literal> PE
+      sections. The stub will try to match the DeviceTrees embedded in the PE image with the DeviceTree
+      provided by the device's firmware by comparing the first compatible string in each UKI DeviceTree
+      with the first compatible string supplied by the firmware. If no match is found the first .dtb
+      section in the image is selected.
```

Comment:
Some markup or explanation is necessary around "compatible". Right now this is very hard to understand.

Also, markup is  missing around `.dtb`.